src/test/java/lotto/domain/LottoTest.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import lotto.strategy.FixtureNumberStrategy;
+import lotto.strategy.NumberStrategy;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoTest {
+
+    @DisplayName("로또 번호 발급")
+    @Test
+    void 로또_번호_발급() {
+        // given
+        NumberStrategy numberStrategy = new FixtureNumberStrategy();
+        // when
+        Lotto lotto = new Lotto(numberStrategy.create(1).get(0).lottoNumbers());
+        // then
+        assertThat(lotto.lottoNumbers()).hasSize(6);
+    }
+
+    @DisplayName("로또 번호 출력문구 생성")
+    @Test
+    void 로또_번호_출력문구_생성() {
+        // given
+        NumberStrategy numberStrategy = new FixtureNumberStrategy();
+        // when
+        Lotto lotto = new Lotto(numberStrategy.create(1).get(0).lottoNumbers());
+        // then
+        assertThat(lotto.toString()).isEqualTo("[2, 3, 5, 6, 10, 40]");
+    }
+
+    @DisplayName("로또 번호 비교")
+    @Test
+    void 로또_번호_비교() {
+        // given
+        NumberStrategy numberStrategy = new FixtureNumberStrategy();
+        Lotto lotto = new Lotto(numberStrategy.create(1).get(0).lottoNumbers());
+        Lotto lottoNumber = new Lotto(List.of(
+                new LottoNumber(1),
+                new LottoNumber(2),
+                new LottoNumber(3),
+                new LottoNumber(4),
+                new LottoNumber(5),
+                new LottoNumber(6)
+        ));
+        // when
+        int result = lotto.match(lottoNumber);
+        // then
+        assertThat(result).isEqualTo(4);
```

Comment:
```suggestion
Lotto lotto = new Lotto(List.of(
                new LottoNumber(10),
                new LottoNumber(2),
                new LottoNumber(3),
                new LottoNumber(40),
                new LottoNumber(5),
                new LottoNumber(6)
        ));
        Lotto lottoNumber = new Lotto(List.of(
                new LottoNumber(1),
                new LottoNumber(2),
                new LottoNumber(3),
                new LottoNumber(4),
                new LottoNumber(5),
                new LottoNumber(6)
        ));
        // when
        int result = lotto.match(lottoNumber);
        // then
        assertThat(result).isEqualTo(4);
```

개인적으로는 테스트 하고자 하는것은 `lotto.match(lottoNumber);`인것 같아요!
그런면에서 `FixtureNumberStrategy`에 대한 의존성이 추가되어야할까? 생각이 들어요~

아니면, `FixtureNumberStrategy.로또넘버_생성함(10,2,3,40,5,6);` 이렇게해서 테스트의 가독성을 증가시켜볼것같아요!

Reply:
실제처럼 자동테스트 번호를 만들고 당첨결과를 비교하는 테스트를 하려고 했습니다.
지금보니 `Lotto lotto = numberStrategy.create(1).get(0);` 로 로또를 받으면 되는데 못보고 넘어갔네요.
변수명도 수정했습니다.